build.sbt

```diff
@@ -20,7 +20,7 @@ lazy val client = project
   .settings(
     name               := "play-soap-client",
     description        := "play-soap client",
-    crossScalaVersions := Seq(scala211, scala212, scala213),
+    crossScalaVersions := Seq(scala212, scala213),
```

Comment:
Dropping Scala 2.11... I hope you are OK with that, but it's outdated anyway (and Play 2.8 does not support it anymore as well)